epub33/overview/index.html

```diff
@@ -237,15 +290,19 @@ <h2>Fixed Layouts</h2>
 						spreads</a> [[EPUB-33]], and <a href="https://www.w3.org/TR/epub-33/#page-spread">how to
 						position pages</a> [[EPUB-33]] within those spreads, providing a broad range of control over the
 					presentation of EPUB Publications.</p>
-			</section>
+
+				<p>The structure and semantics of EPUB Documents with Fixed Layouts are defined in the dedicated <a
+						href="https://www.w3.org/TR/epub-33/#sec-fixed-layouts">section</a> of [[EPUB-33]].</p>
```

Comment:
Accessibility nitpick, but linking the word "section" is ambiguous. This could be:

> The `<a...>`structure and semantics of Fixed-Layout Documents`</a>` are defined in [[EPUB-33]].